nemo_skills/evaluation/evaluator.py

```diff
@@ -262,11 +262,35 @@ def eval_arena(cfg):
             Path(output_file).unlink()
 
 
+@nested_dataclass(kw_only=True)
+class LeanEvaluatorConfig:
+    sandbox: dict = field(default_factory=lambda: {'sandbox_type': 'local'})
+    num_parallel_requests: int = 10
```

Comment:
Does it have a problem if we increase it? I think lean check is quite slow, so if we can run more in parallel, then it's better. But if each lean process itself spans multiple subprocesses, then they will conflict for resources and we might run into timeouts